### ISSUE TITLE
axes format: %y instead of %Y

### ISSUE THREAD

**Issue by yzemaze** [DrFriendless/ExtendedStatsServerless#8]
If values on x-axes where dates instead of numbers setting `"formatType": "time", "format": "%y"` in Vega would work and deliver years without century and therefore a way cleaner look.

**Comment by DrFriendless** [comment]
BTW correct repo for client bugs is https://github.com/DrFriendless/ExtstatsClient.

**Comment by DrFriendless** [comment]
Which charts in particular do you suggest this for?

**Comment by DrFriendless** [comment]
OK, I've updated two charts on the Owned page. I'm a bit torn as the graphs are not using dates as real dates on the X-axis, they really are just bands labelled with the year - if they were really dates, leap years would be 1/365th wider. However I do like the idea of giving Vega more information about the type of data being displayed. Thank you for the suggestion.

**Comment by yzemaze** [comment]
Yep, I had (Plays of) Games Owned by Published Year in mind, but it would be useful for basically every chart with years on x and about 15+ years to display.
You’re welcome. It’s the least I can do to say thank you for years of marvellous work by you for us stats-freaks ;)
(I’ll post there next time.)
